### ISSUE TITLE
Customized Agent creates empty MeshAgent folder

### ISSUE THREAD

**Issue by hunter-nl** [Ylianst/MeshCentral#2151]
When in the config is specified
```
agentCustomization: {
            displayName: Remote Support,
            description: Remote Support agent for remote monitoring, management and assistance.,
            companyName: Company,
            serviceName: CompanyAgent,
            fileName: CompanyAgent
        },
```
Then after, I guess, an upgrade of the agent, an empty folder "MeshAgent" is created in C:\Program Files folder.

**Comment by krayon007** [comment]
The description is not clear. Is the agent installing correctly, then the self update creates an empty folder? I'm not understanding the order of events leading to the empty folder.

**Comment by hunter-nl** [comment]
The customised agent was installed just fine in own customised folder. Today I got an upgrade of the agent and that's also working fine. But an empty MeshAgent folder is also created.

**Comment by yomono** [comment]
I want to expand this:
I had a few hosts with both agents installed: The original, unmodified, and the one customized by me.
(I had a problem with my original server and thats why I had both)

Recently, I updated Meshcentral.
When i did it, This happened:
* A "meshcentral.update" file was created on my customized agent folder (and not deleted, it just stays there)
* The customized agent Service was stopped
* The ORIGINAL service agent started (It could've been started already, can't confirm but is very likely)
* The ORIGINAL "Mesh Agent" folder was updated!

A lot of hosts didn't want to reconnect, so I check them. After I did a "-fulluninstall" command on the original mesh agent, my customized updated correctly and everything went normal again.

At some point, the updater mixed both mesh agent installations.
On @hunter-nl case, maybe nothing happened because he only had the customized agent, and the update only created this empty folder he mentioned.
But in my case, the updater was able to actually update the wrong mesh agent, leaving the customized stopped and unupdated
